src/rust/engine/src/core.rs

```diff
@@ -307,13 +330,17 @@ impl fmt::Display for Failure {
   }
 }
 
+pub fn native_python_traceback(msg: &str) -> String {
+  format!(
+    "Traceback (no traceback):\n  <pants native internals>\nException: {}",
```

Comment:
Is "Traceback (no traceback)" the right string to put here? Looking at this now, this verbiage seems kind of confusing. I'm not sure if this exact string is used by the python interpreter in a way that we want to match, but if not, maybe we should say something like "Traceback (no Python traceback, error in Rust engine code)" or something like that?

Reply:
Yea, possibly... this is existing code though: I just moved it. Once this lands I think we could make another pass at polish for the two tracebacks (and possibly skip rendering the python traceback if we don't have one.)